### ISSUE TITLE
[v0.6.0-rc1] Android: crash on navigation to Send view

### ISSUE THREAD

**Issue by kaloudis** [ZeusLN/zeus#765]
from the Android runtime crash:

```
java.lang.IllegalArgumentException: app.zeusln.zeus: Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent.
02-05 19:20:21.482 11162 11199 E AndroidRuntime: Strongly consider using FLAG_IMMUTABLE, only use FLAG_MUTABLE if some functionality depends on the PendingIntent being mutable, e.g. if it needs to be used with inline replies or bubbles.
```

Extra info:
https://stackoverflow.com/questions/68228666/targeting-s-version-10000-and-above-requires-that-one-of-flag-immutable-or-fl

**Comment by kaloudis** [comment]
Fixed in https://github.com/ZeusLN/zeus/pull/766

**Comment by kaloudis** [comment]
Fixed in v0.6.0-rc2
